package.json

```
@@ -2,7 +2,9 @@
   "name": "@infinite-loop-factory/app-factory",
   "private": true,
   "scripts": {
+    "biome": "npx biome check --fix",
     "clean": "git clean -xdf node_modules",
+    "gen": "pnpm turbo gen",
```

P1

여기 Pnpm  빼도 그대로 동작할 것 같습니다~
```suggestion
    "gen": "turbo gen",
```